installer/Install.ps1

```diff
@@ -48,49 +48,69 @@ Function Test-CommandExist {
 } #end function Test-CommandExist
 
 function Finish {
-    Write-Output "Optimizing VHDX size...."
-    If (Test-CommandExist Optimize-VHD) { Optimize-VHD ".\*.vhdx" -Mode Full }
+    Clear-Host
+    If (Test-CommandExist Optimize-VHD) {
+        Write-Output "Optimizing VHDX size...."
+        Optimize-VHD ".\*.vhdx" -Mode Full
+    }
```

Comment:
I suggest that mounting the VHDXs as read-only before calling Optimize-VHD as it is recommended on Microsoft Learn.
It is said that this can make the outputs even smaller sometimes